operating/manage-compliance/managing-compliance-20.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-compliance-20"]
+= Managing compliance (2.0)
+include::modules/common-attributes.adoc[]
+:context: managing-compliance-20
+
+toc::[]
+
+:FeatureName: Compliance (2.0)
+include::snippets/technology-preview.adoc[]
+
+By navigating to the Compliance (2.0) section in the {rh-rhacs-first} portal, you can view the compliance results associated with your clusters by using the Compliance Operator.
```

Comment:
I am still having a hard time understanding the difference between compliance 1.0/2.0 and the Compliance Operator. Is there a simple way to revise this intro or add additional information to explain?

Reply:
Compliance 2.0 is a new feature that will be introduced in 4.4. Maybe we can bold "Compliance (2.0)", because that's how you see it in the user interface. 
@kcarmichael08 WDYT?

Reply:
I would bold if we are referring to a menu item, and use no bold or parenthesis if just referring to the general feature.

Reply:
```suggestion
You can view the compliance results associated with your cluster by using the compliance 2.0 feature in the {rh-rhacs-first portal}. The feature collects compliance information gathered by the Compliance Operator into a single interface (or something).
```
WDYT? Initially, I was having a hard time understanding the difference between the CO and compliance 2.0. Do you think that this sort of distinction is useful?

If not, I am happy to let this go and not block the PR on this.

Reply:
@dashrews78 @rhmdnd WDYT about this re-wording??

Reply:
I'm OK with it as long as we update `(or something)` to be meaningful.